writing_center/cron/__init__.py

```diff
@@ -47,32 +47,24 @@ def send_reminder_emails(self):
         try:
             upcoming_appointments = self.cron.get_upcoming_appointments()
             for appointment in upcoming_appointments:
-                student = self.cron.get_user(appointment.student_id)
                 tutor = self.cron.get_user(appointment.tutor_id)
+                student = self.cron.get_user(appointment.student_id)
+                subject = "Your Upcoming Appointment"
+
+                appt_info = {'date': appointment.scheduledStart.date().strftime("%m/%d/%Y"),
+                             'time': appointment.scheduledStart.time().strftime("%I:%M %p"),
+                             'tutor': tutor.firstName + ' ' + tutor.lastName}
+
                 if appointment.multilingual:
-                    subject = "Writing Center Multilingual Appointment Reminder"
-                    message = "Thank you for signing up for a multilingual writing support appointment with the " \
-                              "Writing Center. Here are the details of your appointment tomorrow:\n\nTutor: " \
-                              "{0} {1}\nStart Time: {2}\nEnd Time: {3}\nLocation: Writing Center (HC 324)\n\nIf " \
-                              "there's an error or you need to cancel, visit the Writing Center website " \
-                              "(https://tutorlabs.bethel.edu/writing-center), click View Your Appointments, and " \
-                              "click on the appointment to cancel.".format(tutor.firstName, tutor.lastName,
-                                                                           appointment.scheduledStart,
-                                                                           appointment.scheduledEnd)
+                    self.mail.send_message(subject, render_template('emails/upcoming_email_multilingual.html', **locals()), student.email, cc='', bcc='', html=True)
```

Comment:
can we leverage the same template file and just update lines 54-56 to include the differing variables?

Reply:
yeah, i can make both of these changes real quick

Reply:
done